### PR TITLE
Use the icon components directly from the tokens package

### PR DESCRIPTION
**Pull request by sandhose**
This uses the icons directly exported by `compound-design-tokens`, which means we can remove the `svgr` plugin/logic here.

It also includes a few quality of life improvement, like the fact that stories which have icons in them now have *all* the icons instead of just a few

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -24,21 +23,20 @@ export default defineConfig({
...
         "vaul",
         "@floating-ui/react",
-        "@floating-ui/react-dom",
         "@radix-ui/react-context-menu",
         "@radix-ui/react-dropdown-menu",
         "@radix-ui/react-form",
         "@radix-ui/react-separator",
         "@radix-ui/react-slot",
         "@radix-ui/react-tooltip",
+        /^@vector-im\/compound-design-tokens\/.*/,
```
Comment (robintown) [inline vite.config.ts:39]: Can you comment on why this isn't just `"@vector-im/compound-design-tokens"`?

File: src/components/Icon/Gallery.mdx
[the remark this answers is not in the document]
Comment (sandhose) [reply src/components/Icon/Gallery.mdx:1]: Oh it is! Reverted

File: vite.config.ts (hunk 1 above)
Comment (sandhose) [reply vite.config.ts:40]: I added a comment, hopefully clear enough?

**Review by robintown — APPROVED** [review]
Right, looks good!

**Pull request merged**